code/game/machinery/computer/station_alert.dm

```diff
@@ -68,10 +71,13 @@
 				L -= I
 	return !cleared
 
-
 /obj/machinery/computer/station_alert/process()
+	air_alarm_count = LAZYLEN(alarms["Atmosphere"])

```

Comment:
Can you tell me how to reference it then, where do I reference the station alerts?

for(var/stationsalerts/SA in whatever)
   LAZYLEN(SA.alarms["Fire"])?